src/App.css

```diff
@@ -200,7 +200,7 @@ button:focus-visible {
 
 a {
   font-weight: 500;
-  color: white;
+  color: black;
   text-decoration: inherit;
 }
 a:hover {
```

Comment:
Double check your hover colour here. If you have the default css still in it is white for me as I'm on dark mode and it just vanishes
![image](https://github.com/user-attachments/assets/0e407666-7e51-4eb7-9b9d-be57688acbfa)